azurerm/internal/services/netapp/netapp_volume_resource.go

```diff
@@ -351,10 +354,6 @@ func netappVolumeDeleteStateRefreshFunc(ctx context.Context, client *netapp.Volu
 			}
 		}
 
-		if _, err := client.Delete(ctx, resourceGroupName, accountName, poolName, name); err != nil {
-			log.Printf("Error reissuing NetApp Volume %q delete request (Resource Group %q): %+v", name, resourceGroupName, err)
-		}
```

Comment:
(as above) is API deletion working 100% now? previously half of the "delete" calls were ignored/failed silently

Reply:
After merged this PR, only below two test cases would fail. The root cause is that tags doesn't take effect. I've filed an issue on Azure/azure-rest-api-specs#10081. Other all test cases can pass after applied this PR.

 TestAccAzureRMNetAppSnapshot_complete: testing.go:684: Step 0 error: Check failed: Check 2/3 error: azurerm_netapp_snapshot.test: Attribute 'tags.%' expected "1", got "0"
    TestAccAzureRMNetAppSnapshot_update: testing.go:684: Step 0 error: Check failed: Check 3/4 error: azurerm_netapp_snapshot.test: Attribute 'tags.%' expected "1", got "0"